### PR TITLE
Added Club operations lead

### PR DESCRIPTION
**Pull request by abbyfischler**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -266,12 +266,22 @@ func main() {
...
     	{"hired_events_designer.md", "https://github.com/hackclub/jobs/blob/main/directory/events_designer.md"},
+=======
+			{"club-operations-lead.md", "https://github.com/hackclub/jobs/blob/main/directory/club-operations-lead.md"},
+		{"education-engineer.md", "https://github.com/hackclub/jobs/blob/main/directory/education-engineer.md"},
+		{"communications-manager.md", "https://github.com/hackclub/jobs/blob/main/directory/communications-manager.md"},
+    	{"events_designer.md", "https://github.com/hackclub/jobs/blob/main/directory/events_designer.md"},
+		{"philanthropy_position.md", "https://github.com/hackclub/jobs/blob/main/directory/philanthropy_position.md"},
+		{"executive_assistant.md", "https://github.com/hackclub/jobs/blob/main/directory/executive_assistant.md"},
+
+>>>>>>> Stashed changes
```
Comment (kognise) [inline main.go:284]: Looks like there's some Git merge artifacts that might have to be yeeted!

File: directory/club-operations-lead.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
+## Hack Club is hiring a Club Operations Lead.
+
+**Find purpose and passion by supporting technical teens with your customer service and management skills. **As the liaison between teens leading their own after-school Hack Clubs and Hack Club's amazing resources, you'll play a vital role in Hack Club's nonprofit mission to support high school coders in starting coding clubs, hackathons, and building awesome projects.
+
+- **Build relationships, follow up, and provide outstanding customer service**
+- Fun, fast-growing nonprofit tech startup
+- Potential for growth!
```
Comment (jessicard) [inline directory/club-operations-lead.md:7]:
```suggestion
Hack Club is where 16,000+ teenage programmers talk, build, and have fun together. We do crazy things like [hosting AMAs](https://hackclub.com/amas/) with Elon Musk and running [the world's longest hackathon on a train](https://www.youtube.com/watch?v=2BID8_pGuqA).

We are looking for a person to support technical teens with their customer service and management skills.

In this role, you would be the liaison between teens leading their own after-school Hack Clubs and Hack Club's amazing resources. You'll play a vital role in Hack Club's nonprofit mission to support high school coders in starting coding clubs, hackathons, and building awesome projects.

[Visit this job on our website](https://hackclub.com/jobs/club-operations-lead/) for more information.
```

Based off of https://github.com/hackclub/jobs/blob/main/directory/education_engineer.md?plain=1#L3-L9

**Review by jessicard — COMMENTED** [review]
A couple suggestions! I'd tweak this job description to be a more pared down version, kind of like https://github.com/hackclub/jobs/blob/main/directory/tech_lead.md is to https://hackclub.com/jobs/lead-hacker/.

**Review by jessicard — APPROVED** [review]
Nice work! 💪

**Pull request merged**